Assets/MRTK/Core/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -28,18 +28,22 @@ public class MixedRealityToolkitConfigurationProfile : BaseMixedRealityProfile
         #region Mixed Reality Toolkit configurable properties
 
         [SerializeField]
-        [Tooltip("The scale of the Mixed Reality experience.")]
-        private ExperienceScale targetExperienceScale = ExperienceScale.Room;
+        [Tooltip("Experience Settings profile.")]
+        private MixedRealityExperienceSettingsProfile experienceSettingsProfile;
 
         /// <summary>
-        /// The desired the scale of the experience.
+        /// Profile for configuring the experience settings of your project.
+        /// Determines whether your project targers AR/VR, the scale of your experience, and the height of the user where applicable
         /// </summary>
-        public ExperienceScale TargetExperienceScale
+        public MixedRealityExperienceSettingsProfile ExperienceSettingsProfile
         {
-            get { return targetExperienceScale; }
-            set { targetExperienceScale = value; }
+            get { return experienceSettingsProfile; }
+            internal set { experienceSettingsProfile = value; }
         }
 
+        [Obsolete("Please use the ExperienceSettingsProfile.TargetExperienceScale")]
+        public ExperienceScale TargetExperienceScale => ExperienceSettingsProfile.IsNull() ? ExperienceScale.Room : ExperienceSettingsProfile.TargetExperienceScale;
```

Comment:
Should we leave the previously serialized `targetExperienceScale` around (and deprecate it) to maintain back compat instead of defaulting to always returning `ExperienceScale.Room` here? This pattern might break users if they changed their profile from the default.